docker-deploy/training_template/docker-compose-eggroll.yml

```diff
@@ -161,6 +162,7 @@ services:
       - fateflow
     networks:
       - fate-network
+    command: ["bash", "-c", "flow init --ip ${FATE_FLOW_IP} --port ${FATE_FLOW_PORT} && pipeline init --ip ${FATE_FLOW_IP} --port ${FATE_FLOW_PORT} && jupyter notebook --ip=0.0.0.0 --port=20000 --allow-root --debug --NotebookApp.notebook_dir='/data/projects/fate/' --no-browser --NotebookApp.token='' --NotebookApp.password='${NOTEBOOK_HASHED_PASSWORD}'"]
```

Comment:
Should be just 

```
["bash", "-c", jupyter notebook --ip=0.0.0.0 --port=20000 --allow-root --debug --NotebookApp.notebook_dir='/data/projects/fate/' --no-browser --NotebookApp.token='' --NotebookApp.password='${NOTEBOOK_HASHED_PASSWORD}'"]
``` 
?

Because the image definition has already had run the other commands.

Reply:
Yes，I checked the command in container. There's no `flow init xxx`, only `jupyter notebook xx`. I didn't understand why.